scripts/tests/test_zkai_native_seq32_attention_mlp_single_proof_gate.py

```
@@ -0,0 +1,154 @@
+    def test_mutation_inventory_is_exact(self) -> None:
+        result = self.__class__.payload["mutation_result"]
+        self.assertTrue(result["all_mutations_rejected"])
+        self.assertEqual(result["mutations_rejected"], len(self.gate.MUTATION_NAMES))
+        self.assertEqual(tuple(result["mutation_names"]), self.gate.MUTATION_NAMES)
+        self.assertEqual(tuple(case["name"] for case in result["cases"]), self.gate.MUTATION_NAMES)
+
+    def test_rejects_mutation_case_name_drift(self) -> None:
+        payload = copy.deepcopy(self.__class__.payload)
+        payload["mutation_result"]["cases"][0]["name"] = "different"
+        payload["payload_commitment"] = self.gate.payload_commitment(payload)
+        with self.assertRaisesRegex(
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)